device-types/Nokia/7750-SR-1.yaml

```diff
@@ -0,0 +1,45 @@
+---
+manufacturer: Nokia
+model: 7750 SR-1
+slug: nokia-7750-sr-1
+part_number: 3HE13571CA
+airflow: front-to-rear
+u_height: 2
+is_full_depth: false
+weight: 33.0 # empty
+weight_unit: lb
+front_image: true
+rear_image: true
+console-ports:
+  - name: Console
+    type: rj-45
+power-ports:
+  - name: PS1
+    type: iec-60320-c14
+  - name: PS2
+    type: iec-60320-c14
```

Comment:
Are these PSU's modular? If so, they should be modelled as `module-bays` on this device, and the associated PSU modules should be created in the module-types folder if they do not already exist

If not, just let me know and mark this resolved

Reply:
Yes you are right. I will add a module-bay for this (based on official documentation). But again I don't know where to find the part numbers, sorry.